packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -464,6 +476,7 @@ export default function TranslationView() {
                         }}
                         onFocus={() => setSidebarWordIndex(i)}
                         onShowDetail={() => setShowSidebar(true)}
+                        onOpenNotes={() => setShouldOpenNotes(true)}
```

Comment:
Here you can just execute `sidebarRef.current.openNotes()` directly. No need to create additional state and effects.

Reply:
Done. I ended up wrapping this call in a `setTimeout` as well for the case where the user does not have a sidebar open (so `sidebarRef.current` is `null`)